### DISCUSSION TITLE
String options in Android xml

### DISCUSSION THREAD

**Discussion by KwadroNaut** [WeblateOrg/weblate#10117] [Q&A]
We have strings like `<string name="stats_up" translatable="false">` which is good, but I would like to also have the option to add something like `comment for translators` Sometimes I have screenshots, but not always.

**Comment by Geeyun-JY3** [comment]
Do you mean https://developer.android.com/guide/topics/resources/localization#provide-context-strings? 
https://docs.weblate.org/en/latest/formats.html#x

**Comment by nijel** [comment]
Just add a XML comment above the string:

```xml
   <!-- comment -->
   <string name="id">String</string>
```
